albumentations/augmentations/transforms.py

```diff
@@ -107,6 +109,7 @@ def __init__(
         min_width: Optional[int] = 1024,
         pad_height_divisor: Optional[int] = None,
         pad_width_divisor: Optional[int] = None,
+        position: str = "center",
```

Comment:
Maybe would be better to use `Enum` for position type? You could describe it as a class parameter.

Reply:
@Dipet added, can you check, please?